dae/dae/annotation/score_annotator.py

```diff
@@ -277,23 +284,29 @@ def _fetch_substitution_scores(self, allele):
             self.get_scores()
         )
 
-    def _fetch_aggregated_scores(self, chrom, pos_begin, pos_end):
-        scores_agg = self.score.fetch_scores_agg(
-            chrom,
-            pos_begin,
-            pos_end,
-            self.get_scores(),
-            self.non_default_position_aggregators,
-            self.non_default_nucleotide_aggregators
-        )
-        if scores_agg is None:
-            return None
+    def _collect_score_queries(self) -> list[ScoreQuery]:
+        result: list[ScoreQuery] = []
+        for attr in self.get_annotation_config():
+            result.append(NPScoreQuery(
+                attr["source"], attr.get("position_aggregator"),
+                attr.get("nucleotidy_aggregator")))
```

Comment:
Potential typo - maybe you meant `nucleotide_aggregator`?

Reply:
Fixed 😄